BricksAndHearts.UnitTests/ControllerTests/Landlord/LandlordControllerTests.cs

```diff
@@ -24,34 +21,8 @@ public void RegisterGet_CalledByUnregisteredUser_ReturnsRegisterViewWithEmail()
             .Which.Email.Should().Be(unregisteredUser.GoogleEmail);
     }
 
-    [Fact]
-    public void AddNewPropertyPost_CalledByUnregisteredUser_Returns403()
-    {
-        // Arrange 
-        CreateUnregisteredUserInController(_underTest);
-        var formResultModel = A.Fake<PropertyViewModel>();
-
-        // Act
-        var result = _underTest.AddNewProperty(formResultModel) as StatusCodeResult;
-
-        // Assert
-        result!.StatusCode.Should().Be(403);
-    }
+    /*

```

Comment:
Can this commented out code be deleted (assuming it's just been moved elsewhere)?

Reply:
Yes! Thanks for spotting that 😅